static/js/weather-components.js

```
@@ -157,6 +308,95 @@ function calculateWetbulbTemp(tempF, humidity) {
+
+// Helper function to get severe weather alert message
+function getSevereWeatherMessage(weatherData) {
+    if (!weatherData || !weatherData.current) return '';
+
+    const current = weatherData.current;
+    const icon = current.icon;
+    const windSpeed = current.wind_speed || 0;
+    const precipRate = current.precipitation_rate || 0;
+    const uvIndex = current.uv_index || 0;
+
+    let messages = [];
```

Biome (1.9.4)

[error] This let declares a variable that is only assigned once.

'messages' is never reassigned.

Safe fix: Use const instead.


(lint/style/useConst)